client/src/context/interface/Profile.ts

```diff
@@ -0,0 +1,31 @@
+import { User } from './User';
+
+export interface Profile {
+  _id: string;
+  isDogSitter: { type: Boolean; require: true; default: false };
+  isAvailable: { type: Boolean; require: true; default: false };

```

Comment:
This is a bit confusing. Why not just allow `isDogSitter: boolean;`. Require and defaults are all handled by Mongo but in terms of the data we pass down, it just have to be the type set on the model. So `isDogSitter: boolean;` still gives the same behaviour and more understandable.